app/src/main/java/com/pajato/android/gamechat/exp/model/Checkers.java

```diff
@@ -0,0 +1,185 @@
+package com.pajato.android.gamechat.exp.model;
+
+import com.google.firebase.database.Exclude;
+import com.google.firebase.database.IgnoreExtraProperties;
+import com.pajato.android.gamechat.exp.ExpType;
+import com.pajato.android.gamechat.exp.Experience;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.pajato.android.gamechat.exp.ExpType.checkers;
+
+/** Provide a Firebase model class for a checkers game experience. */
+@IgnoreExtraProperties
```

Comment:
I have just noticed that a lot of classes that follow use these annotations that model objects for Firebase.  I could be wrong but I believe that there should be a single high level abstraction (TicTacToe, Chess, Checkers) for an experience that has these annotations (the experience model class) and that contained object classes should not have the annotations.  I don't know that it does harm but it is an observation worth making.